catalogue/webapp/components/IIIFViewer/IIIFViewer.tsx

```diff
@@ -223,32 +255,76 @@ const IIIFViewer: FunctionComponent<IIIFViewerProps> = ({
     shouldScrollToCanvas = true,
     query = '',
   } = fromQuery(router.query);
-  const [gridVisible, setGridVisible] = useState(false);
+  const [gridVisible, setGridVisible] = useState(false); // TODO put back to false - then change how tabs are done
```

Comment:
Can you explain the comment as the value is currently `false`?

Reply:
just forgot to remove the comment